src/mon/PGMap.cc

```diff
@@ -2621,7 +2621,9 @@ void PGMap::get_health_checks(
 
   // REQUEST_SLOW
   // REQUEST_STUCK
-  if (cct->_conf->mon_osd_warn_op_age > 0 &&
+  // SLOW_OPS unifies them in mimic.
+  if (osdmap.require_osd_release < CEPH_RELEASE_MIMIC &&
```

Comment:
But below we still use REQUEST_SLOW/ REQUEST_STUCK as the warn-titles...

Reply:
@xiexingguo i don't follow you. if you need more context, please see #18614.

Reply:
e.g.:
```c++
     if (!warn_detail.empty()) {
       ostringstream ss;
       ss << warn << " slow requests are blocked > "
 	 << cct->_conf->mon_osd_warn_op_age << " sec";
auto& d = checks->add("REQUEST_SLOW", HEALTH_WARN, ss.str());
```

We still use "REQUEST_SLOW" as the warn-type

Reply:
again, could you read John's comment in #18614 , where `SLOW_OPS` unifies these two warnings. so after mimic, they will vanish, but before that, pre-mimic OSDs should be able to report to mgr.

Reply:
Ahh, right. Thanks for the explanation, I got it now